builder/context/buildcontext.go

```diff
@@ -116,7 +117,8 @@ func getRunnableDirs(cwd string) ([]RunnableDir, bool, error) {
 		// determine if a .runnable file exists in that dir
 		innerFiles, err := ioutil.ReadDir(dirPath)
 		if err != nil {
-			return nil, false, errors.Wrapf(err, "failed to list files in %s", tf.Name())
+			log.Printf("couldn't read files in %v", dirPath)
```

Comment:
There are some logging functions that most of `subo` uses (in the `util` package), could you create a `LogWarn` function in there and use it here, for consistency?

Reply:
@cohix Done, as a good Rubyist I've also DRY'd up the log util :)